tasks/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -50,21 +56,36 @@ spec:
         set -eux
 
         SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
-        COMPONENTS_LENGTH=$(jq '.components |length' ${SNAPSHOT_PATH})
+        COMPONENTS_LENGTH=$(jq '.components |length' "${SNAPSHOT_PATH}")
```

Comment:
same comment about linting ticket

Reply:
Same answer - I'm only fixing the parts that are being changed.